test/models/Author.spec.js

```diff
@@ -0,0 +1,85 @@
+import assert from 'power-assert';
+import mockgoose from 'mockgoose';
+import Promise from 'bluebird';
+import AutherModel from 'models/Author';
+
+describe('/models/Author', ()=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).